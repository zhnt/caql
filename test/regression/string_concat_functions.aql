// Test string concatenation in functions
function make_greeting(name) {
    return "Hello, " + name + "!";
}

function make_message(prefix, suffix) {
    return prefix + " world " + suffix;
}

print(make_greeting("Alice"));
print(make_greeting("Bob"));

print(make_message("Hello", "today"));
print(make_message("Good", "morning"));

// Function returning concatenated strings
function get_info(id) {
    return "ID: " + string(id) + ", Status: Active";
}

print(get_info(100));
print(get_info(200));
