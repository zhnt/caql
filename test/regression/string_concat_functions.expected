Hello, Alice!
Hello, Bob!
Hello world today
Good world morning
ID: 100, Status: Active
ID: 200, Status: Active
